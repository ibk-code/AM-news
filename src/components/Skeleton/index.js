import React, { Component, Fragment } from 'react';

class Skeleton extends Component {
  render() {
    return (
      <Fragment>
        <div className="skeleton-wrap mb-4">
          <div className="skeleton-shape"></div>
          <div className="skeleton-shape-long mt-4"></div>
          <div className="skeleton-shape-small mt-3"></div>
        </div>
      </Fragment>
    );
  }
}

export default Skeleton;
