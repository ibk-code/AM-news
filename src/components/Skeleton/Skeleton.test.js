import { shallow } from 'enzyme';
import React from 'react';
import Skeleton from './index';

describe('Skeleton Component test', () => {
  it('expect to render skeleton component', () => {
    expect(shallow(<Skeleton />)).toMatchSnapshot();
  });
});
