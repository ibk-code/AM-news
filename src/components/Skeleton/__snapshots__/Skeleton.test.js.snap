// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton Component test expect to render skeleton component 1`] = `
<Fragment>
  <div
    className="skeleton-wrap mb-4"
  >
    <div
      className="skeleton-shape"
    />
    <div
      className="skeleton-shape-long mt-4"
    />
    <div
      className="skeleton-shape-small mt-3"
    />
  </div>
</Fragment>
`;
